File: Week_08/Javascript/file.js
// function eat(food){
// 	console.log("Eating some" + food);
// }

var exampleArray = [ "pizza",
					"hot dog",
					"pizza",
					"ice cream",
					"fish",
					"steak",
					"oranges"]
					; 

function findOccurrences(array, wordToLookFor){
	var totalOccurrences = 0;
	
	array.forEach(function(word){
		if (word === wordToLookFor){
			totalOccurrences++;
		}
	});
	
	return totalOccurrences
}

var count = findOccurrences(exampleArray, "pizza");
console.log(count);

var foods = [ 'pizza', 'cookies', 'bread' ];

// foods.forEach(function (food) {
//     console.log(food);
// });

// var capFoods = foods.map(function(food)){
// 	return food.toUppercase();

// }

// console.log(capFoods)

// var foodMsg = foods.reduce(function (pre, food) {
//     return pre + ' and ' + food;
// });

// console.log(foodMsg);

// var breadless = foods.filter(function(food)){
// 	return food !== "Bread";
// }

// var food = "Pizza";

// if (food === "Pizza"){
// 	console.log("Pizza is my favorite food too");
// } else if (food === "Sushi"){
// 	console.log("Sushi is pretty good...");
// } else {
// 	console.log("Thats wierd...");
// }

var myQuote = "The dog says 'Bark' "

console.log(myQuote)

File: Week_08/Javascript/file.rb
def eat(food)
	puts "Eating some #{food}"
end

eat "Pizza"

def find_occurrences
end

i = 1

while i <= 42
	puts i
	i += 1
end

foods = [ 'pizza', 'cookies', 'bread' ]

caps_foods = foods.map do | food |
	food.upcase
end

p caps_foods

nums = [2, 6 , 7, 42]

total = nums.reduce do | sum, num |
	 sum + num
end

breadless = foods.select do
	food != "Bread"
end

food = "Pizza"

if food == "Pizza"
	puts ""
elsif food == "Sushi"
	puts ""
else
	puts ""
end

def add(a, b)